### ISSUE TITLE
Incorrect role set on AccordionItem component

### ISSUE THREAD

**Issue by dakahn** [IBM/carbon-components-react#2251]
The accordion items have a `role="presentation"' which is causing an accessibility violation with Axe.

### references
https://dequeuniversity.com/rules/axe/3.2/list
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role
